apps/custom-ui-floating-window-demo/README.md

```diff
@@ -0,0 +1,57 @@
+![4](https://user-images.githubusercontent.com/19341550/207391343-dbd28993-64e7-4a5d-8613-5ef15d8980df.gif)
+
+We can split the configuration and installation steps into three stages. Let's get it running. ✨
+
+### Stage 1: Cloning the App, installing dependencies and making it publicly reachable
+
+1. Clone the repository and switch to the app directory using the `cd` command.
+
+2. After you have switched to the app directory using the terminal, install the required dependencies using the `npm install` command.
+
+3. The app makes use of Prisma ORM to provide data storage. Based on the schema we have defined in `prisma/schema.prisma` you can create an actual database using the following command `npx prisma db push`.
+
+```
+% npx prisma db push
+Environment variables loaded from .env
+Prisma schema loaded from prisma/schema.prisma
+Datasource "db": SQLite database "dev.db" at "file:./dev.db"
+
+SQLite database dev.db created at file:./dev.db
+
+🚀  Your database is now in sync with your Prisma schema. Done in 28ms
+
+✔ Generated Prisma Client (4.6.1 | library) to ./node_modules/@prisma/client in 58ms
+```
+
+4. This command will create an SQLite database at `prisma/dev.db`.
+
+5. The app will run on port 3000. For the floating window to work, the app URL should be publicly accessible. Use ngrok to expose port 3000 using the `ngrok http 3000` command.
+
+6. Now that we have performed the core steps in stage 1, the next step is to create an app in Pipedrive and add a floating window.
+
+### Stage 2: Creating a Pipedrive App with Floating Window App Extension.
+
+1. Follow the app creation [steps mentioned in the documentation](https://pipedrive.readme.io/docs/marketplace-creating-a-proper-app#create-an-app-in-5-simple-steps) but make sure that it is a private app (since it is for demo purposes only).
+2. At the minimum, you must provide the app name and callback URL, which is `http://localhost:3000/api/auth/callback`.
+3. Since we are creating a floating window app, we need to define the right scopes and the iframe URL in addition to the minimum required fields for a private app.
+4. Make sure you select the following scopes - Deals (Full Access), Activities (Full Access), Contacts (Full Access), Search for all data and Phone calls integration.
+5. Navigate to the App Extensions section and click on the `+ Custom Floating Window` button under the Custom UI Extensions subsection.
+6. Provide a name for the window along with the description and iframe URL. This URL is the one generated by ngrok, and it looks like this: `https://example.ngrok.io/`.
+7. Make sure you choose the appropriate entry point option. Once everything is filled, click on the `Add Floating Window` button. You can also [find more details related to this step](https://pipedrive.readme.io/docs/custom-ui-extensions-floating-window#how-can-i-add-a-custom-floating-window-in-marketplace-manager) in the documentation.
```

Comment:
Can you here mention to specifically choose the "Phone number and Calls tab" entry point?

Reply:
Sure Andreas. Let me add them